prysm/x/polarization.py

```diff
@@ -273,31 +286,108 @@ def linear_polarizer(theta=0, shape=None):
 
     return linear_diattenuator(0, theta=theta, shape=shape)
 
+def vector_vortex_retarder(charge, shape, retardance=np.pi, theta=0):
```

Comment:
taking `shape` as an input is not prysm-like; please take appropriate coordinate arrays and do not make a grid internally

Reply:
I’ll have it take in an angular coordinate `theta` and update the tutorial to include it